app/build.gradle

```diff
@@ -172,8 +172,8 @@ def getDate() {
 }
 
 android {
-    compileSdkVersion "Google Inc.:Google APIs:24"
-    buildToolsVersion "25.0.0"
+    compileSdkVersion 26
```

Comment:
Per some online resources https://stackoverflow.com/questions/35450417/errorcause-failed-to-find-target-with-hash-string-google-inc-google-apis23 http://laudevapps.weebly.com/blog/errorfailed-to-find-target-with-hash-string-google-incgoogle-apis25 it looks like the longer version has been deprecated (if you try 25 or 26 with the longer string the hash doesn't resolve). Very Google-y though, I can find no mention of this in official documentation.